packages/dx-react-grid/docs/reference/grid.md

```diff
@@ -53,6 +53,15 @@ Field | Type | Description
 ------|------|------------
 children? | ReactElement | A React element to be placed in the footer.
 
+## Plugin Components
+
+Name | Properties | Description
+-----|------------|------------
+Grid.Root | [GridRootProps](#gridrootprops) | A component that renders the grid root layout.
+Grid.Header | [GridHeaderPlaceholderProps](#gridheaderplaceholderprops) | A component that renders the grid header placeholder.
```

Comment:
Why do we call a component that used as a default for the `headerPlaceholderComponent` property as 'Header'?